pyproject.toml

```
@@ -0,0 +1,74 @@
+[project]
+classifiers=[
+  "Operating System :: OS Independent",
+  "Intended Audience :: Science/Research",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  'Operating System :: Microsoft :: Windows',
+  'Operating System :: Unix',
+  'Operating System :: MacOS',
+  "Topic :: Scientific/Engineering",
+]
+
+dynamic = ["version"]
+keywords = ["unstructured grids"]
+description = "Xarray-styled package for reading and directly operating on unstructured grid datasets following UGRID conventions"
+license = {file = "LICENSE.txt"}
+name = "uxarray"
+readme = "README.md"
+requires-python = ">=3.9"
+
+# minimal dependencies start
+dependencies = [
+  "antimeridian",
+  "cartopy",
+  "cartopy",
+  "dask[dataframe]",
+  "datashader",
+  "geoviews",
+  "holoviews",
+  "matplotlib-base",
+  "matplotlib-inline",
+  "netcdf4",
+  "numba",
+  "numpy",
+  "pandas<2.1.0",
+  "pyarrow",
+  "requests",
+  "scikit-learn",
+  "scipy",
+  "shapely",
+  "spatialpandas",
+  "xarray",
+]
+# minimal dependencies end
+
+[project.optional-dependencies]
+complete = ["uxarray[math, dev]"]
+math = ['gmpy2', 'pyfma']
+dev = ['pathlib', 'pre_commit', 'pytest', 'pytest-cov', 'ruff']
+
+[project.urls]
+Documentation = "uxarray.readthedocs.io/"
+Source = "https://github.com/UXARRAY/uxarray"
+Tracker = "https://github.com/UXARRAY/uxarray/issues"
+
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools>=60",
+  "setuptools-scm>=8",
+]
+
+[tool.setuptools]
+packages = ["uxarray"]
```

```suggestion
[tool.setuptools.packages]
find = {}
```

This gets rid of the packaging warnings while maintaining the end result of the internal package structure.

---

Thanks for the fix!

---

I think you missed applying this one

---

Good catch! I forgot to push it.